xtc/src/ledger.rs

```diff
@@ -39,22 +39,24 @@ impl Ledger {
 
     #[inline]
     pub fn deposit(&mut self, account: Principal, amount: u64) {
-        StatsData::deposit(amount);
+        let fee = 2_000_000_000;
```

Comment:
In rust is it typical to pull consts to the top as a named variable?

Reply:
Yes, it is.

Reply:
Sorry I meant is it not typical to pull it right to the top of the file?